VirtualEconomyFramework/VEDriversLite/Neblio/NeblioTransactionHelpers.cs

```diff
@@ -437,7 +437,7 @@ public static (Transaction, double) GetTransactionWithNeblioInputs(ICollection<U
         /// <param name="tutxos">Optional input token utxo</param>
         /// <param name="fee">Fee - 20000 minimum</param>
         /// <returns>New Transaction Hash - TxId</returns>
-        public static async Task<string> MintNFTTokenAsync(MintNFTData data, EncryptionKey ekey, ICollection<Utxos> nutxos, ICollection<Utxos> tutxos)
+        public static async Task<Transaction> MintNFTTokenAsync(MintNFTData data, EncryptionKey ekey, ICollection<Utxos> nutxos, ICollection<Utxos> tutxos)

```

Comment:
Could you please implement this for Dogecoin drivers in DogecoinTransactionHelpers?

Reply:
Sure Tomas will create a new Issue to work on DogecoinTransactionHelpers.cs

Reply:
Sure Tomas.